hoomd/mpcd/update.py

```diff
@@ -0,0 +1,132 @@
+# Copyright (c) 2009-2024 The Regents of the University of Michigan.
+# Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+r"""MPCD reverse perturbation to establish nonequilibrium shear flow.
+
+These are MPCD updaters.
```

Comment:
This is a module that potentially holds all updaters, not just the RNES one. So, the module docstring should be more general. This is sufficient.
```suggestion
r"""MPCD updaters.
```